src/components/facet-browser/helper.ts

```diff
@@ -130,7 +130,7 @@ export const getFacetFieldTranslation = (name: FacetField) => {
     case FacetField.WorkTypes:
       return "facetWorkTypesText";
     default:
-      return invalidSwitchCase<string>(name);
+      return invalidSwitchCase<string>(name as never);
```

Comment:
Why was `never` needed?
Would be nice with a comment.

Reply:
The API specs changed, and it just didn't work all of a sudden. So I just followed the specs. No idea why it needs never.